source/base/symmetric_tensor.cc

```diff
@@ -138,5 +147,10 @@ const unsigned int SymmetricTensor<rank,dim,Number>::n_independent_components;
 
 #include "symmetric_tensor.inst"
 
+namespace internal
+{
+#include "symmetric_tensor.inst2"
```

Comment:
I would have put the opening and closing of the namespace into the `.inst.in` file. (I find that conceptually simpler because then each file is sort of self-contained.) Or do we use this idiom elsewhere as well?